== TEST (2019-03-14 19:36:03) 
-- CODE:
aresta =5
area =round(3/2*(3)**0,5*5**2, 4)
 
print(round(area, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    area =round(3/2*(3)**0,5*5**2, 4)
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 19:37:07) 
-- CODE:
aresta =5
area =round(3/2*(3)**0,5*5**2, 4)
 
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    area =round(3/2*(3)**0,5*5**2, 4)
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 19:37:25) 
-- CODE:
aresta =5
area =round(3/2*(3)**0,5*5**2, 4)
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    area =round(3/2*(3)**0,5*5**2, 4)
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 19:37:31) 
-- CODE:
aresta =6
area =round(3/2*(3)**0,5*5**2, 4)
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    area =round(3/2*(3)**0,5*5**2, 4)
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 19:37:41) 
-- CODE:
aresta =6
area =round(3/2*(3)**0,5*5**2, 4)

print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    area =round(3/2*(3)**0,5*5**2, 4)
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 19:38:30) 
-- CODE:
aresta =6
area =3/2*(3)**0,5*5**2

print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 4, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-14 19:38:35) 
-- CODE:
aresta =6
area =3/2*(3)**0,5*5**2

print(round(3/2*(3)**0,5*5**2, 4))
-- EXECUTION TIME:
0.206299
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1577/1332/main.py", line 4, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 19:38:39) 
-- CODE:
aresta =6
area =3/2*(3)**0,5*5**2

print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 4, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 19:38:57) 
-- CODE:
aresta =5cm
area =3/2*(3)**0,5*5**2

print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
File "XXXX", line 1
    aresta =5cm
              ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 19:39:09) 
-- CODE:
aresta =5
area =3/2*(3)**0,5*5**2

print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 4, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 19:40:09) 
-- CODE:
aresta = 6
area =3/2*(3)**0,5*5**2

print(round(3/2*(3)**0,5*5, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 4, in <module>
    print(round(3/2*(3)**0,5*5, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 22:46:04) 
-- CODE:
print(round(3/2*(3)**0,5**5, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5**5, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-14 22:46:21) 
-- CODE:
print(round(3/2*(3)**0,5**5, 4))
-- EXECUTION TIME:
0.174863
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1577/1332/main.py", line 1, in <module>
    print(round(3/2*(3)**0,5**5, 4))
TypeError: round() takes at most 2 arguments (3 given)

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 22:46:38) 
-- CODE:
aresta = 6
area =3/2*(3)**0,5*5**2
print(round(3/2*(3)**0,5**5, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    print(round(3/2*(3)**0,5**5, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 22:46:44) 
-- CODE:
aresta = 6
area =3/2*(3)**0,5*5**2
print(round(3/2*(3)**0,5**5, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    print(round(3/2*(3)**0,5**5, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-14 22:46:55) 
-- CODE:
aresta = 6
area =3/2*(3)**0,5*5**2
print(round(3/2*(3)**0,5**5, 4))
-- EXECUTION TIME:
0.194
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1577/1332/main.py", line 3, in <module>
    print(round(3/2*(3)**0,5**5, 4))
TypeError: round() takes at most 2 arguments (3 given)

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 22:47:49) 
-- CODE:
aresta = 6
area =3/2*(3)**0,5*5**2
print(round(3/2*(3)**0.5**5, 4))
-- OUTPUT:
1.5524
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-14 22:47:57) 
-- CODE:
aresta = 6
area =3/2*(3)**0,5*5**2
print(round(3/2*(3)**0.5**5, 4))
-- EXECUTION TIME:
0.131495
-- TEST CASE 1:
---- input:
xxx
---- correct output:
64.9519
---- user output:
1.5524
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 22:48:54) 
-- CODE:
aresta = 6
area =3/2*(3)**2*5**2
print(round(3/2*(3)**0.5**5, 4))
-- OUTPUT:
1.5524
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-14 22:49:01) 
-- CODE:
aresta = 6
area =3/2*(3)**2*5**2
print(round(3/2*(3)**0.5**5, 4))
-- EXECUTION TIME:
0.17654
-- TEST CASE 1:
---- input:
xxx
---- correct output:
64.9519
---- user output:
1.5524
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 22:49:50) 
-- CODE:
aresta = 6
area =3/2*(3)**2*5**2
print(round(3/2*(3)**2*5**2, 4))
-- OUTPUT:
337.5
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-14 22:49:58) 
-- CODE:
aresta = 6
area =3/2*(3)**2*5**2
print(round(3/2*(3)**2*5**2, 4))
-- EXECUTION TIME:
0.111406
-- TEST CASE 1:
---- input:
xxx
---- correct output:
64.9519
---- user output:
337.5
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 22:50:57) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-14 22:51:02) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- EXECUTION TIME:
0.132168
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1577/1332/main.py", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 22:52:41) 
-- CODE:
Aresta=6
Area= print(3/2*(3)**0,5*5**2)
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-14 22:52:56) 
-- CODE:
Aresta=6
Area= print(3/2*(3)**0,5*5**2)
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 22:04:34) 
-- CODE:
print(round(3/2*(3)**0,5*6**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*6**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 22:04:37) 
-- CODE:
print(round(3/2*(3)**0,5*6**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*6**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 22:06:02) 
-- CODE:
print(round(3/2*(3)**0,5*6*6, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*6*6, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 22:06:49) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-18 22:06:53) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- EXECUTION TIME:
0.125805
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1577/1332/main.py", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 22:14:49) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 23:39:07) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 23:39:35) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-18 23:39:40) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- EXECUTION TIME:
0.118165
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1577/1332/main.py", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 23:45:50) 
-- CODE:
print(round(3/2*(3)**0,5*5**2*6, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2*6, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 23:50:13) 
-- CODE:
print(round(3/2*(3)**0,5*5**2*, 4))
-- ERROR:
File "XXXX", line 1
    print(round(3/2*(3)**0,5*5**2*, 4))
                                  ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 23:51:03) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-18 23:51:11) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- EXECUTION TIME:
0.136962
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1577/1332/main.py", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 23:51:43) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 23:52:07) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 23:52:57) 
-- CODE:
print(round(3/2*(3)**0,5* 5** 2  , 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5* 5** 2  , 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 23:58:15) 
-- CODE:
aresta= 5
area= 64.95190528
print(round(3/2*(3)**0,5* 5** 2  , 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    print(round(3/2*(3)**0,5* 5** 2  , 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 23:58:24) 
-- CODE:
aresta= 5
area= 64.95190528

print(round(3/2*(3)**0,5* 5** 2  , 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 4, in <module>
    print(round(3/2*(3)**0,5* 5** 2  , 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-18 23:58:43) 
-- CODE:
aresta= 5
area= 64.95190528

print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 4, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-19 00:01:47) 
-- CODE:
aresta= 5
area= 64.95190528

print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 4, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-19 00:01:52) 
-- CODE:
aresta= 5
area= 64.95190528

print(round(3/2*(3)**0,5*5**2, 4))
-- EXECUTION TIME:
0.147672
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1577/1332/main.py", line 4, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-19 00:02:37) 
-- CODE:
aresta= 5
area= 64.95190528

print(round(3/2*(3)**0,5*6*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 4, in <module>
    print(round(3/2*(3)**0,5*6*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-19 00:02:53) 
-- CODE:
aresta= 5
area= 64.95190528

print(round(3/2*(3)**0,5*6*5**2, 4))
-- EXECUTION TIME:
0.111325
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1577/1332/main.py", line 4, in <module>
    print(round(3/2*(3)**0,5*6*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-19 00:04:02) 
-- CODE:
print(round(3/2*(3)**0,5*6*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*6*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-19 00:04:09) 
-- CODE:
print(round(3/2*(3)**0,5*6*5**2, 4))
-- EXECUTION TIME:
0.152394
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1577/1332/main.py", line 1, in <module>
    print(round(3/2*(3)**0,5*6*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-19 00:04:36) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-19 00:05:00) 
-- CODE:
print(round(3/2*(3)**0,5*6*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*6*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-19 00:05:25) 
-- CODE:
print(round(3/2*(3)**0,5*5**2, 4))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-19 00:13:04) 
-- CODE:


print(round(3/2*(3)**0,5*5**2, 4))

-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-19 00:15:22) 
-- CODE:


print(round(3/2*(3)**0,5*5**2, 4))

-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 3, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-19 00:17:23) 
-- CODE:

print(round(3/2*(3)**0,5*5**2, 4))

-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    print(round(3/2*(3)**0,5*5**2, 4))
TypeError: round() takes at most 2 arguments (3 given)
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-19 00:19:19) 
-- CODE:


print(round(3/2*(3)**0.5*5**2, 4))
-- OUTPUT:
64.9519
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-19 00:20:12) 
-- CODE:


print(round(3/2*(3)**0.5*5**2, 4))
-- EXECUTION TIME:
0.160313
-- TEST CASE 1:
---- input:
xxx
---- correct output:
64.9519
---- user output:
64.9519
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
